clang/test/Analysis/malloc.cpp

```diff
@@ -214,3 +214,15 @@ void *realloc(void **ptr, size_t size) { realloc(ptr, size); } // no-crash
 namespace pr46253_paramty2{
 void *realloc(void *ptr, int size) { realloc(ptr, size); } // no-crash
 } // namespace pr46253_paramty2
+
+namespace pr81597{
+struct S {};
+struct T {
+	void free(const S& s);
+};
+void f(T& t) {
+	S s;
+    // This is not the free you are looking for...
+	t.free(s); // no-warning
+}
+} // namespace pr81597
```

Comment:
```suggestion
namespace pr81597 {
struct S {};
struct T {
  void free(const S& s);
};
void f(T& t) {
  S s;
  t.free(s); // no-warning: This is not the free you are looking for...
}
} // namespace pr81597
```
Indent by 2 spaces, have a space before the `{` token.